jupyterhub/app.py

```diff
@@ -329,11 +329,10 @@ def _load_classes(self):
             load_roles = [
                             {
                                 'name': 'teacher',
-                                'description': 'Access users information, servers and groups without create/delete privileges',
+                                'description': 'Access to users' information, servers and groups',
```

Comment:
The `users` scope doesn't include servers anymore, so we should either change the description or add the `users:servers` scope here (I know it's only example for config but better make it consistent I think).

Reply:
Good point, updated the information